pimd/pim_cmd.c

```diff
@@ -6861,6 +6878,28 @@ DEFUN (no_ip_pim_packets,
 	return CMD_SUCCESS;
 }
 
+DEFPY (igmp_group_watermark, igmp_group_watermark_cmd,
+       "[no$no] ip igmp watermark-warn [(10-60000)$limit]",
```

Comment:
`$no` is unnecessary, it's bound automatically, see the docs http://docs.frrouting.org/projects/dev-guide/en/latest/cli.html#type-rules

Reply:
Thanks for this tip.
For now split into 2 commands as per your suggestion. So made as mandatory in other command